qa/supply-chain/config.toml

```diff
@@ -4,15 +4,22 @@
 [imports.bytecode-alliance]
 url = "https://raw.githubusercontent.com/bytecodealliance/wasmtime/main/supply-chain/audits.toml"
 
+[imports.chromeos]
+url = "https://chromium.googlesource.com/chromiumos/third_party/rust_crates/+/main/cargo-vet/audits.toml?format=TEXT"
+
+[[imports.chromeos.criteria-map]]
+ours = "crypto-reviewed"
+theirs = "crypto-safe"
```

Comment:
Technically I think there's a difference in review standard. By the chromeos conventions, if a crate doesn't *itself* implement crypto primitives, it can be marked "does-not-implement-crypto" which means "Inspection reveals that the crate in question does not attempt to implement any cryptographic algorithms on its own." This implies "crypto-safe" but doesn't imply that the crate has been cryptographically reviewed.

Our "crypto-reviewed", on the other hand, always requires cryptographic review, because to us that means that the crate *using the intended crypto correctly* (in particular, according to the intended specification).

Reply:
Hmm, I saw them as equivalent, because implementing a cryptographic algorithm is not the same thing as implementing a cryptographic primitive. I interpret our `crypto-reviewed` policy as "we've reviewed everything up to the cryptographic protocols", but that doesn't extend to e.g. serialization code that uses cryptographic code internally. But if ChromeOS does in fact mean a different "API boundary" than us, then we can remove this.